src/DigitalTwin/pnp/src/common/utility.ts

```diff
@@ -94,44 +114,61 @@ export class Utility {
    * @param modelId model id
    * @param content model content
    */
-  // tslint:disable-next-line:no-any
-  static async createModelFile(folder: string, modelId: string, content: any): Promise<void> {
-    const type: ModelType = DeviceModelManager.convertToModelType(content[DigitalTwinConstants.TYPE]);
+  static async createModelFile(
+    folder: string,
+    modelId: string,
+    // eslint-disable-next-line  @typescript-eslint/no-explicit-any
```

Comment:
do we need "no-explicit-any" comment? tslint allows type of any

Reply:
Yes. eslint does not allow any.